Jasmine/Common/Constants.swift

```diff
@@ -78,6 +78,14 @@ enum Constants {
 
             static let tileFallInterval: TimeInterval = 0.5
         }
+
+        enum GameOver {
```

Comment:
Maybe I have misunderstood what you said, but nope overall it's the same. It's either I store the control statements here or in the VC itself. So nope.